<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QDialog" name="frmLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>111</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>登录 - 洛仙都</string>
  </property>
  <widget class="QLabel" name="lbUsername">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>QQ号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>39</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>本系统密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnUsername">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnPassword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnJoinus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>免登录</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRemember">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住我</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
